api/paylike_Transations.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+* Team Paylike
+*
+*  @author    Team Paylike
+*  @copyright Team Paylike
+*  @license   MIT license: https://opensource.org/licenses/MIT
+*/
+
+include_once(dirname(__FILE__).'/paylike_Subsystem.php');
+class PaylikeTransactions extends PaylikeSubsystem
+{
+	public function fetch($id_transaction)
+	{
+		return $this->request('GET', '/transactions/'.$id_transaction);
+	}
+
+	public function voids($id_transaction, $opts)
```

Comment:
I think this method should be named `void` when the others are named `capture` etc.
If this is not possible due to "void" being a reserved word, I think "cancel" is a better substitute.